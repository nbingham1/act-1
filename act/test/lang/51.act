defproc foo(bool x, y)
{
  bool w;
  prs {
   x <5,lvt> -> x-
   w -> x-

   passn<5,lvi,lvt>(x,x,y)
  }
}

foo f;
