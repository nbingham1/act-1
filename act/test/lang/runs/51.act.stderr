ERROR: File `51.act', line 8, col 15
	The identifier `lvi' does not exist in the current scope
